index.html

```diff
@@ -211,6 +211,23 @@ <h1>About me</h1>
         </div>
       </div>
     </section>
-      
+    <div class="contact-me">
+      <h2 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/meqdprbd" class="contact-form" method="post">
+        <input type="text" class="contact-form-text" maxlength="30" placeholder="Full name" required>
+        <input type="email" class="contact-form-email" placeholder="Email address" required>
+        <textarea name="contact" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
+        <div class="form-button"><input type="submit" value="Get In Touch" class="contact-button"></div>

```

Comment:
- [ ] Kindly align the submit button to the center and try to match the design in Figma (`font-size` and `padding`) 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200628790-cb40c931-2457-4537-853f-19f63e674468.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200628826-f1670dae-1e61-4f65-add6-300ff6fc6c4f.png)